src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -9,7 +10,7 @@ class EngineFunctions(val ctx: TaskExecutionContext) extends WdlFunctions {
       // TODO: Validate parameters
 
       // Just assume the first parameter is a String (example only)
-      val path = params.head.value.asInstanceOf[String]
+      val path = params.head.asInstanceOf[WdlStringValue].value
```

Comment:
if params is empty this will blow up. You can make path an Option[WdlStringValue] by something like:

    params.headOption.map(_.asInstanceOf[WdlStringValue].value)